verilog/tools/ls/verilog_ls.cc

```diff
@@ -146,7 +151,24 @@ int main(int argc, char *argv[]) {
   // -- Register JSON RPC callbacks
 
   // Exchange of capabilities.
-  dispatcher.AddRequestHandler("initialize", InitializeServer);
+  dispatcher.AddRequestHandler(
+      "initialize",
+      [&symboltablehandler](const verible::lsp::InitializeParams &p) {
+        // TODO add filling the include_paths and corpus
+        std::cerr << "Initialize workspace paths:  rootPath='" << p.rootPath
```

Comment:
Tip: you can use `LOG(INFO)` to output logging without necessarily spilling it into.
(you can influence what is logged by setting the `VERIBLE_LOGTHRESHOLD` environment variable)

Reply:
I replaced `std::cerr` in the current code with `LOG(...)` where possible, I'll keep that in mind in future commits